man/tpm2_print.1.md

```diff
@@ -0,0 +1,51 @@
+% tpm2_print(1) tpm2-tools | General Commands Manual
+%
+% DECEMBER 2018
+
+# NAME
+
+**tpm2_print**(1) - Prints TPM data structures
+
+# SYNOPSIS
+
+**tpm2_print** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_print**(1) decodes a TPM data structure and prints enclosed
+elements to stdout as YAML.
+
+# OPTIONS
+
+  * **-t**, **--type**:
```

Comment:
Is there anyway we could figure this out for the user? I didn't see anything as a dead giveaway...

Reply:
We could do something like assume type=TPMS_ATTEST when the first 4 bytes (magic) are ff544347, but that's a bit flaky. I'm also not sure that every TPM generated object uses the same magic. We can always add this later without breaking the API.